### PR TITLE
updated event info for May

### PR DESCRIPTION
**Pull request by Jmartimus**

### CODE REVIEW

File: src/utils/constants/events.js [hunk 1]
```diff
@@ -334,11 +343,11 @@ export const pastEventData = [
...
   {
     imageBackgroundColor: null,
-    date: 'April 13, 2021',
+    date: 'May 11, 2021',
     imageLink:
       'https://images.unsplash.com/photo-1588702547923-7093a6c3ba33?ixid=MXwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHw%3D&ixlib=rb-1.2.1&auto=format&fit=crop&w=1950&q=80',
-    meetupLink: 'https://www.meetup.com/ReactJSDallas/events/276279684/',
-    speaker1: "Tim Waite - How to choose the right JS framework - And why it's probably React",
+    meetupLink: 'https://www.meetup.com/ReactJSDallas/events/mrkxmrycchbpb/',
+    speaker1: 'Adrianne Mallett - "Writing Tests First"',
```
Comment (patrickhulce) [inline src/utils/constants/events.js:350]: aw dang this sounds great, hate to miss it

File: src/utils/constants/events.js [hunk 2]
```diff
@@ -1,4 +1,13 @@
 export const pastEventData = [
+  {
+    imageBackgroundColor: null,
+    date: 'April 13, 2021',
```
Comment (patrickhulce) [inline src/utils/constants/events.js:4]: heads up the previous events just use <month> <year>

might be worth matching for consistency :)

**Review by patrickhulce — APPROVED** [review]
thanks!

File: src/utils/constants/events.js (hunk 2 above)
Comment (Jmartimus) [reply src/utils/constants/events.js:4]: Sorry, I do not understand.

Did you mean the date format?

Comment (patrickhulce) [reply src/utils/constants/events.js:4]: oh whoops, sorry github treated them as HTML, edited :)

Yes I'm suggesting `April 2021`
```suggestion
    date: 'April 2021',
```

Comment (Jmartimus) [reply src/utils/constants/events.js:4]: Ah, last month when Joe and I updated it, we decided to start including the actual day as well.  I guess we should have put something about changing the format.

Comment (patrickhulce) [reply src/utils/constants/events.js:4]: ah ok, nevermind then carry on! :)

Comment (Jmartimus) [reply src/utils/constants/events.js:4]: Thanks!

File: src/utils/constants/events.js (hunk 1 above)
Comment (MightyJoeW) [inline src/utils/constants/events.js:350]: If the double quotes around the title weren't intentional, I'd remove them to keep the style consistent with the other talk titles.

Comment (Jmartimus) [reply src/utils/constants/events.js]: Just redid it and pushed it.

**Pull request merged**